react-app/src/ui/components/three/HexagonBackground.tsx

```diff
@@ -67,6 +69,17 @@ export const HexagonBackground = ({
     setSelectedHex,
   } = useStateStore();
 
+
+  //Any comments on how I am using dojo to read squad positions is appreciated.
+  const positions = useEntityQuery([Has(Position)]).map((entity) => {
+    const positions = useComponentValue(Position, entity);
+    return positions;
+  });
+  //See of selected hex is on a squad position
+  const isOnSquad = positions.filter(p => p?.game_id == 5).some((pos) => {
+    pos?.x === selectedHex?.col && pos?.y === selectedHex?.row;
+  });
+  
```

Comment:
isOnSquad is not used atm so selecting anywhere shows the bounds. What is the coordinate system for squads? I see x : 1000 and y : 1000

![Screenshot 2024-02-06 at 12 40 09 PM](https://github.com/cartridge-gg/hegemony/assets/95137353/ad53ef47-8c25-416f-9132-6bdc7c5a3694)

Reply:
the first squad spawns at 1000,1000. We just offset in the client so we don't underflow in cairo

Reply:
I see offset as 
`export const offset = 995;`
is this right?